portal-impl/src/com/liferay/portal/service/impl/ResourceLocalServiceImpl.java

```diff
@@ -453,6 +453,17 @@ public boolean hasUserPermissions(
 		return hasUserPermissions;
 	}
 
+	@Override
+	public void removeResourcePermission(
+			String actionId, AuditedModel auditedModel, long roleId, int scope)
+		throws PortalException {
+
+		_resourcePermissionLocalService.removeResourcePermission(
+			auditedModel.getCompanyId(), auditedModel.getModelClassName(),
+			scope, String.valueOf(auditedModel.getPrimaryKeyObj()), roleId,
+			actionId);
```

Comment:
I already know that on this we will have problems 😢 because the params order is different but if we align them they will not be alphabetically sorted.